lib/rspec/bash/matchers/called_with_arguments.rb

```diff
@@ -16,4 +17,21 @@
 
     called_with_correct_args && called_correct_number_of_times
   end
+
+  failure_message do |actual_command|
+    command_name = actual_command.command
+
+    formatted_expected_call = "#{command_name} #{expected_argument_list.join(' ')}"
+    formatted_actual_calls = actual_command.call_log.call_log_arguments.map do |arg_array|
+      "#{command_name} #{arg_array.join(' ')}"
+    end.join("\n")
```

Comment:
Should the logic for
`actual_command.call_log.call_log_arguments.map do |arg_array|
  "#{command_name} #{arg_array.join(' ')}"
end.join("\n")`
be hidden away in the CallLog? I'm not actually sure, but worth thinking about